Makefile

```diff
@@ -1,10 +1,30 @@
-all:
-	go get github.com/tools/godep
+CWD:=$(shell pwd)
+RUNC_TEST_IMAGE=runc_test
+PROJECT=github.com/opencontainers/runc
+TEST_DOCKERFILE=test_Dockerfile
+
+all: godep
 	godep go build -o runc .
```

Comment:
Isn't this `godep` invocation here literally _just_ to set `GOPATH` to include `_workspace`?  Couldn't we just change this `all` target to be simply the following and only download/install/use `godep` for people who actually need to manage the directory contents?
```Makefile
all:
    GOPATH="$$GOPATH:$$PWD/Godeps/_workspace" go build -o runc .
```

Or even more:
```Makefile
export GOPATH=$(GOPATH):$(CURDIR)/Godeps/_workspace

all:
    go build -o runc .

# .....
```